app/_includes/md/enterprise/turn-on-rbac.md

```diff
@@ -25,7 +25,7 @@ If you have a Docker installation, run the following command to set the needed e
 
 **Note:** make sure to replace `<kong-container-id>` with the ID of your container.
 
-<pre><code>echo "KONG_ENFORCE_RBAC=on \<br>KONG_ADMIN_GUI_AUTH=basic-auth \<br>KONG_ADMIN_GUI_SESSION_CONF='{\"secret\":\"secret\",\"storage\":\"kong\",\"cookie_secure\":false}' \<br>kong reload exit" | docker exec -i <div contenteditable="true">{KONG_CONTAINER_ID}</div> /bin/sh</code></pre>
+<pre><code>echo "KONG_ENFORCE_RBAC=on <br>KONG_ADMIN_GUI_AUTH=basic-auth <br>KONG_ADMIN_GUI_SESSION_CONF='{\"secret\":\"secret\",\"storage\":\"kong\",\"cookie_secure\":false}' <br>kong reload exit" | docker exec -i {KONG_CONTAINER_ID}/bin/sh</code></pre>
```

Comment:
You can actually just turn this into a normal codeblock now, with backticks. The html tags were only needed because of the editable content.

There is also a space missing here:
```
{KONG_CONTAINER_ID}/bin/sh
```
Should be:
```
{KONG_CONTAINER_ID} /bin/sh
```